kyuubi-hive-jdbc/src/main/java/org/apache/kyuubi/jdbc/hive/KyuubiStatement.java

```diff
@@ -543,6 +543,18 @@ public int getMaxRows() throws SQLException {
     return maxRows;
   }
 
+  @Override
+  public boolean getMoreResults(int current) throws SQLException {
+    // getMoreResults() javadoc says, that it should implicitly close
+    // any current ResultSet object, so here in case of Statement.CLOSE_CURRENT_RESULT
+    // we can implement the same logic
+    // see also https://issues.apache.org/jira/browse/HIVE-7680
+    if (current == Statement.CLOSE_CURRENT_RESULT) {
+      return false;
```

Comment:
Hmm, should we perform `result.close` here?

Reply:
I think there might be reasons why this wasn't added in the `HIVE-7680` fix, but it seems like we can actually close the resultSet here. Added resultSet closing.